src/app/header/header.component.html

```diff
@@ -81,7 +81,6 @@ <h5 id="offcanvasStartLabel">penda</h5>
         ></button>
     </div>
     <hr />
-    <br />
```

Comment:
Was this removed for a reason?

Reply:
Yes, because the transition in that area ( above ''Cilesimet'' ) was taking a bit more time, all the time.